src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/AllocationsMigrationResource.kt

```diff
@@ -124,4 +117,95 @@ class AllocationsMigrationResource(
       includeOnlyFailures = includeOnlyFailures,
     ),
   )
+
+  @PreAuthorize("hasRole('ROLE_MIGRATE_ACTIVITIES')")
+  @GetMapping("/allocations/history/{migrationId}")
```

Comment:
Should the top level declaration be changed to /migrate/allocations to easily accommodate new endpoints?

Reply:
It wouldn't hurt would it. I've mainly just done a copy and paste job from another migration but I'll do that in next PR.